What's new article (#29232)
dotnet/docs: docs/whats-new/dotnet-docs-2022-04-01.md

---

> title: ".NET docs: What's new for April 1, 2022 - April 30, 2022"
> description: "What's new in the .NET docs for April 1, 2022 - April 30, 2022."

Reviewer: Still haven't made this code change:

```suggestion
title: ".NET docs: What's new for April 2022"
description: "What's new in the .NET docs for April 2022."
```

ms.date: 05/01/2022
---

# .NET docs: What's new for April 1, 2022 - April 30, 2022

> Welcome to what's new in the .NET docs from April 1, 2022 through April 30, 2022. This article lists some of the major changes to docs during this period.

Reviewer: same comment:

```suggestion
Welcome to what's new in the .NET docs for April 2022. This article lists some of the major changes to docs during this period.
```

## .NET Framework

### New articles

> - [\<disableStackOverflowProbing> Element](../framework/configure-apps/file-schema/runtime/disablestackoverflowprobing-element.md)

Reviewer: @gewarren This is the 2nd build verifier warning. Was this file moved?

## .NET fundamentals

### New articles

- [Debug ThreadPool Starvation](../core/diagnostics/debug-threadpool-starvation.md)
- [Diagnostic port](../core/diagnostics/diagnostic-port.md)
- [Multi-level lookup is disabled](../core/compatibility/deployment/7.0/multilevel-lookup.md)
- [SYSLIB0041: Some Rfc2898DeriveBytes constructors are obsolete](../fundamentals/syslib-diagnostics/syslib0041.md)
- [SYSLIB0042: FromXmlString and ToXmlString on ECC types are obsolete](../fundamentals/syslib-diagnostics/syslib0042.md)
- [SYSLIB0043: ECDiffieHellmanPublicKey.ToByteArray is obsolete](../fundamentals/syslib-diagnostics/syslib0043.md)
- [MVC's detection of an empty body in model binding changed](../core/compatibility/aspnet-core/7.0/mvc-empty-body-model-binding.md)
- [C++/CLI projects in Visual Studio](../core/compatibility/core-libraries/7.0/cpluspluscli-compiler-version.md)
- [CA1851: Possible multiple enumerations of `IEnumerable` collection](../fundamentals/code-analysis/quality-rules/ca1851.md)

### Updated articles

- [dotnet-coverage code coverage utility](../core/additional-tools/dotnet-coverage.md) - Adding new options and example scenarios

## C# language

> ### New articles
> 
> - [Title not found in: #28890](../csharp/language-reference/operators/null-parameter-check.md)
> 

Reviewer: This article was removed in #29103 

```suggestion
```

### Updated articles

- [Built-in reference types (C# reference)](../csharp/language-reference/builtin-types/reference-types.md) - Document raw string literals
- [Strings and string literals](../csharp/programming-guide/strings/index.md) - Document raw string literals

## Community contributors

The following people contributed to the .NET docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).

- [GitHubPang](https://github.com/GitHubPang) (7)
- [nschonni](https://github.com/nschonni) - Nick Schonning (3)
- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (3)
- [bvda](https://github.com/bvda) - Brian Vestergaard Danielsen (1)
- [cartermp](https://github.com/cartermp) - Phillip Carter (1)
- [chessboards](https://github.com/chessboards) (1)
- [DreadLordMikey](https://github.com/DreadLordMikey) - Michael Hofer (1)
- [Flash0ver](https://github.com/Flash0ver) - Stefan Pölz (1)
- [ftsfranklin](https://github.com/ftsfranklin) (1)
- [gfoidl](https://github.com/gfoidl) - Günther Foidl (1)

> - [github-actions](https://github.com/github-actions) -  (1)

Reviewer: bot:
```suggestion
```
